src/database/prepare.ts

```diff
@@ -2,9 +2,11 @@
 // loads all the environment variables from a .env file
 // @see https://github.com/motdotla/dotenv/tree/v6.1.0#how-do-i-use-dotenv-with-import
 import * as dotenv from "dotenv";
-dotenv.config();
-
 import * as path from "path";
+dotenv.config({
+  path: path.resolve(process.cwd(), ".env.example")
```

Comment:
@alexgpeppe could you please give me some context on this edit? What is supposed to do and why? BTW, I don't like having a file name hardcoded here, but depends by the reason. That's why I'm asking :)

Reply:
The _.env.example_ file contains the definitions for all the environment variables required by the app. When the app is run with docker-compose, the env vars are correctly read and set from that file thanks to the configuration within the _docker-compose.yaml_ file; when the app is run without docker-compose, the dotenv library takes care of reading and setting the env vars within the app.
By default, dotenv searches for a _.env_ file in the working directory of the running process (in our case, the project root folder). If this file is located somewhere else or has a different name, then one must provide dotenv with the full path, so that the library knows where to read the env vars from, as described in its [documentation](https://github.com/motdotla/dotenv#path).

Reply:
So, my error, I'd suggest to roll back the change and use .env. SORRY! :(